front/public/locales/en/operationalStudies/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "affairCode": "Affair code",
+  "budget": "Budget",
+  "createdOn": "Created on",
+  "createStudy": "Create a study",
+  "description": "description",
+  "error": {
+    "unableToPostDocumentTitle": "Picture upload failed",
+    "unableToPostDocument": "Picture loading impossible"
+  },
+  "filterPlaceholder": "Search…",
+  "fundedBy": "Funding",
+  "geremiCode": "GEREMI code",
+  "modifiedOn": "Updated on",
+  "modifyProject": "Update project",
+  "objectives": "objectives",
+  "openStudy": "Open",
+  "projectBudget": "Estimated budget",
+  "projectCancel": "Cancel",
+  "projectCreateButton": "Create the projet",
+  "projectCreationTitle": "Create a project",
+  "projectDeleteButton": "Delete",
+  "projectDeleted": "Successful suppression!",
+  "projectDeletedDetails": "Project {{ name }} has been deleted.",
+  "projectDescription": "Project description",
+  "projectFunders": "Funding details",
+  "projectModificationTitle": "Modify project",
+  "projectModifyButton": "Modify",
+  "projectName": "Project name",
+  "projectNameMissing": "You have to give a name to the project",
+  "projectObjectives": "Project objectives",
+  "projectObjectivesMarkdown": "Formatted objectives",
+  "projectsList": "Projects",
```

Comment:
Seen this a couple of times, and it's not on you, but we should remove the `s` here: `projectList`. Lots of other variables are incorrectly named likeso.

Reply:
You're perfectly right, my bad, I putted this... everywhere (for studies, scenarios, too), a lot of change to do... and to test, so to do later (could-you eventually create an issue for this? thx!)